### PR TITLE
feat: add `math/base/special/atand`

### LINKED ISSUE #37: Implement `atand`
Compute the arctangent of `x` where the output is in degrees. `@stdlib/math/base/special/atand`.

### PR DESCRIPTION
**Pull request by saisrikardumpeti**
Resolves #37 

## Description

> What is the purpose of this pull request?

This pull request:

-  Adds functionality to compute `atand` of any given input.

| input | atand |
| -------- | ------- |
| 0  | 0 |
| $1 \div \sqrt{3}$ |  30.0   |
| 1 | 45.0  |
| $\sqrt{3}$ | 60 |
| Infinity | 90.0 |

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by saisrikardumpeti** [comment]
@Planeshifter this PR is ready for review.

File: lib/node_modules/@stdlib/math/base/special/atand/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "gypfile": true,
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/atand/package.json]:
```suggestion
```

File: lib/node_modules/@stdlib/math/base/special/atand/package.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    "stdlib",
+    "stdmath",
+    "mathematics",
+    "math",
+    "math.asin",
+    "asin",
+    "degree",
+    "arctangent",
+    "sine",
+    "inverse",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/atand/package.json]:
```suggestion
    "degree",
    "arctangent",
    "tangent",
    "inverse",
```

File: lib/node_modules/@stdlib/math/base/special/atand/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+var v = atand( 0.0 );
+// returns 0.0
+
+v = atand( 0.5 );
+// returns ~26.57
+
+v = atand( 1 / sqrt( 3 ) );
+// returns ~30.0
+
+v = atand( 1 );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/atand/README.md]:
```suggestion
v = atand( 1.0 / sqrt( 3.0 ) );
// returns ~30.0

v = atand( 1.0 );
```

File: lib/node_modules/@stdlib/math/base/special/atand/docs/repl.txt [hunk 4]
```diff
@@ -0,0 +1,26 @@
+
+{{alias}}( x )
+    Computes the arctangent of a double-precision floating-point number.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/atand/docs/repl.txt]:
```suggestion
    Computes the arctangent (in degrees) of a double-precision floating-point
    number.
```

File: lib/node_modules/@stdlib/math/base/special/atand/lib/index.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+* var v = atand( 0.0 );
+* // returns 0.0
+*
+* var v = atand( 0.5 );
+* // returns ~26.57
+*
+* var v = atand( 1 / Math.sqrt( 3 ) );
+* // returns ~30.0
+*
+* var v = atand( 1 );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/atand/lib/index.js]:
```suggestion
* var v = atand( 1.0 / Math.sqrt( 3.0 ) );
* // returns ~30.0
*
* var v = atand( 1.0 );
```

File: lib/node_modules/@stdlib/math/base/special/atand/lib/main.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+* @example
+* var v = atand( 0.5 );
+* // returns ~26.57
+*
+* @example
+* var v = atand( 1 / Math.sqrt( 3 ) );
+* // returns ~30.0
+*
+* @example
+* var v = atand( 1 );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/atand/lib/main.js]:
```suggestion
* var v = atand( 1.0 / Math.sqrt( 3.0 ) );
* // returns ~30.0
*
* @example
* var v = atand( 1.0 );
```

**Review by Planeshifter — APPROVED** [review]

**Comment by Planeshifter** [comment]
Thank you for the PR! Let's get this one in.

**Pull request merged**